stubs/stripe/stripe/api_resources/abstract/api_resource.pyi

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+from stripe import api_requestor as api_requestor, error as error, util as util
```

Comment:
I wonder if these `util as util` imports can be removed? I suspect that they are internal and thus don't need to be exposed. They are generated in many files.

Reply:
Yes, this is a stubgen quirk; it is overzealous in generating re-exports. We ran into this with a few other packages recently too but I haven't looked into whether there's a concrete change we can make.

Reply:
I was contemplating only generating re-exports in `__init__.pyi` or if there is an explicit re-export in the implementation, but so far I haven't found time to work on this.